src/main/java/org/jenkinsci/plugins/junitrealtimetestreporter/RealtimeJUnitStep.java

```diff
@@ -403,6 +417,13 @@ public Descriptor<?> getDelegateDescriptor() {
             return Jenkins.getInstance().getDescriptor(JUnitResultArchiver.class);
         }
 
+        // Hack: using the delegate pattern with a <st:include> Jelly element doesn't
+        // seem to pick up doFillXxxItems methods automatically, so we have to explicitly
+        // delegate to the JUnitResultArchiver's descriptor class
+        public ListBoxModel doFillStdioRetentionItems(@QueryParameter("stdioRetention") String value) {
+            return new JUnitResultArchiver.DescriptorImpl().doFillStdioRetentionItems(value);
```

Comment:
I thought this would be cleaner as well, but it doesn't compile because the `Descriptor` interface doesn't include the `doFillStdioRetentionItems` method.

Reply:
There was another method which casts it for you automatically I pushed

Reply:
Ah, nice! Thanks.